apps/keyboard/js/views/swipeable_panel_view.js

```diff
@@ -0,0 +1,260 @@
+'use strict';
+
+/* global EmojiKeyView, KeyView */
+
+(function(exports) {
+
+/**
+ * SwipeablePanelView is used to render a panel that is swipeable to go through
+ * several sections.
+ */
+
+function SwipeablePanelView(layout, options, viewManager) {
```

Comment:
Since BaseView would receive `target` as its first parameter while 'SwipeablePanelView' needs layout here, so I think it is fine here.